docs/content/installation/helm.md

```diff
@@ -90,13 +90,26 @@ PGO will check for updates upon startup and once every 24 hours. Any errors in c
 
 ## Upgrade and Uninstall
 
-And once PGO has been installed, it can then be upgraded and uninstalled using applicable `helm`
-commands:
+Once PGO has been installed, it can then be upgraded using the `helm upgrade` command.
+However, before running the `upgrade` command, any Custom Resource Definitions (CRDs) must first be
+manually updated (this is specifically due to a
+[design decision in Helm v3](https://helm.sh/docs/chart_best_practices/custom_resource_definitions/),
```

Comment:
⛏️ This link describes how to design a chart with CRDs. One of these seems like a better fit for chart users:

- https://helm.sh/docs/topics/charts/#custom-resource-definitions-crds
  Describes CRDs a bit and how it works when there's a custom resource in the templates.

- https://helm.sh/docs/topics/charts/#limitations-on-crds
  Gets right to it and says what Helm does _not_ do; upgrades nor uninstalls.

I like the latter, but maybe that's just me.

```suggestion
[design decision in Helm v3](https://helm.sh/docs/topics/charts/#limitations-on-crds),
```